index.html

```diff
@@ -0,0 +1,146 @@
+
+
+            <p>2022.02.18(SAT)- 20(MON) </p>
+            <p>Gran Salon Corferias, Bogotá-Colombia</p>
+
+
+
+        </section>
+        <section id="program" class="program">
+
+            <div>Main Program</div>
+
+            <div class="program_out">
+                <div class="program_in">
+                    <div><img src="./images/icon_01.png" alt="title" /></div>
+                    <p>Lecture</p>
+                    <h4> We will present global trends and developments and feature articles of interest that reflect
+                        the current state of design.</h4>
+                </div>
+                <div class="program_in">
+                    <div> <img src="./images/icon_02.png" alt="title" /></div>
+                    <p>Exhibition</p>
+                    <h4>Exhibition of works in different finishes by Colombian leads who have participated in different
+                        fairs around the world.
+                    </h4>
+
+                </div>
+                <div class="program_in">
+                    <div><img src="./images/icon_03.png" alt="title" /></div>
+                    <p>Forums</p>
+                    <h4> Designers from various contries speaking about their experiences and new trends.</h4>
+                </div>
+                <div class="program_in">
+                    <div><img src="./images/icon_04.png" alt="title" /></div>
+                    <p>Worshop</p>
+                    <h4> We will do a practical workshop, where we will explore the most used techniques for design
+                        thinking.</h4>
+                </div>
+                <div class="program_in">
+                    <div><img src="./images/icon_05.png" alt="title" /></div>
+                    <p>CHPT Feedback</p>
+                    <h4> We will make a small feedback in figures of these years of activity and how the design movement
+                        has grown and we will see how it has grown.</h4>
+                </div>
+
+
+            </div>
+            <div>SEE</div>
+        </section>
+
+
+
+        <section class="speaker_heros" id="speakers">
+
+            <div class="centered-title">
+                <h2 class="black-title">Featured Heros</h2>
+                <span class="line"></span>
+            </div>
+
+            <div class="heros-container"></div>
+
+            
+
+        </section>
+
+        <section id="sponsors" class="sponsors">
+
+           <h2 class="white-tittle">Sponsors</h2>
+        <div class="sponsor-row">
+            <div><img src="./images/google.png" alt="logo" /></div>
+            <div><img src="./images/autodesk.png" alt="logo" /></div>
+            <div><img src="./images/avengers.png" alt="logo" /></div>
+            <div><img src="./images/makita.png" alt="logo" /></div>
+            <div><img src="./images/stark.png" alt="logo" /></div>
+            </div>
+        </section>
+        
+    </main>
+
+    <div class="footer-home">
```

Comment:
- [ ] You can use more semantic HTML in your code, for example, consider using ```<footer>``` tag here instead of ```<div>```, this will help your website white the SEO and your code easy to maintain

Reply:
I have a doubt because i had the same css for the both file htm index and about i use footer tag for about.html and the ndex.html (home) i used div for this last by no have conflct,  i add image  i will appreciate you suggestion thanks
![image](https://user-images.githubusercontent.com/92118813/169573890-7e0f3383-d695-4a72-8eb0-bf26b6011a43.png)

Reply:
Hello @diegoh40 You can use the same footer markup you used in the about for the index. This is to ensure that you use semantic tags